include/mull/Config/ConfigParser.h

```diff
@@ -149,6 +149,8 @@ template <> struct MappingTraits<mull::RawConfig> {
     io.mapOptional("cache_directory", config.cacheDirectory);
     io.mapOptional("junk_detection", config.junkDetection);
     io.mapOptional("parallelization", config.parallelizationConfig);
+    io.mapOptional("mutation_testing_elements_report_path",
+                   config.mutationTestingElementsReportPath);
```

Comment:
You would also need to add this to the `mull-cxx`.

Reply:
Could you explain what you are referring to?

Reply:
I don't see how to configure it via `mull-cxx`.